packages/scandipwa/src/component/MyAccountOverlay/MyAccountOverlay.component.js

```diff
@@ -185,16 +185,17 @@ export class MyAccountOverlay extends PureComponent {
               isCheckout={ isCheckout }
               setLoadingState={ setLoadingState }
               onSignIn={ onSignIn }
+              isLoading={ false }
```

Comment:
What is the point in this case? Should it be always false? Maybe then add default prop to MyAccountSignIn if it is not passed, then be false?